CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `custom/text`: The `content` setting and all its properties are deprecated in favor of `format` with the same functionality. ([`#2676`](https://github.com/polybar/polybar/pull/2676))
 
 ### Added
+-  Support for actions on hover via `mousebtn::HOVER_START` and `mousebtn::HOVER_END` ([`#1064`](https://github.com/polybar/polybar/issues/1064), [`#2868`](https://github.com/polybar/polybar/pull/2868)).
+-  `custom/script`: `hover-start` and `hover-end` actions ([`#1064`](https://github.com/polybar/polybar/issues/1064), [`#2868`](https://github.com/polybar/polybar/pull/2868)).
 -  Added support for TAG_LABEL (`<label>`) in ipc module  ([`#2841`](https://github.com/polybar/polybar/pull/2841))  by [@madhavpcm](https://github.com/madhavpcm).
 -  Added support for format-i for each hook-i defined in ipc module ([`#2775`](https://github.com/polybar/polybar/issues/2775),  [`#2810`](https://github.com/polybar/polybar/pull/2810))  by [@madhavpcm](https://github.com/madhavpcm).
```

Comment:
There should only be a single space after the dash (the two entries below are a mistake).

```suggestion
- Support for actions on hover via `mousebtn::HOVER_START` and `mousebtn::HOVER_END` ([`#1064`](https://github.com/polybar/polybar/issues/1064), [`#2868`](https://github.com/polybar/polybar/pull/2868)).
- `custom/script`: `hover-start` and `hover-end` actions ([`#1064`](https://github.com/polybar/polybar/issues/1064), [`#2868`](https://github.com/polybar/polybar/pull/2868)).
- Added support for TAG_LABEL (`<label>`) in ipc module  ([`#2841`](https://github.com/polybar/polybar/pull/2841))  by [@madhavpcm](https://github.com/madhavpcm).
- Added support for format-i for each hook-i defined in ipc module ([`#2775`](https://github.com/polybar/polybar/issues/2775),  [`#2810`](https://github.com/polybar/polybar/pull/2810))  by [@madhavpcm](https://github.com/madhavpcm).
```